XML/Other/ReplaceableTestParameters.xml

```diff
@@ -278,7 +278,7 @@ Scenario 2 : You need to run all the performance tests quickly.
 	</Parameter>
 	<Parameter>
 		<ReplaceThis>INFINIBAND_IBM_NBC_TESTNAMES</ReplaceThis>
-		<ReplaceWith>allreduce</ReplaceWith>
+		<ReplaceWith></ReplaceWith>
```

Comment:
those are useless?

Reply:
Those are new change in Intel Benchmark MPI, 2019 version